core/src/main/java/io/confluent/rest/Application.java

```diff
@@ -452,6 +452,13 @@ protected void configureSecurityHandler(ServletContextHandler context) {
       context.setSecurityHandler(createBasicSecurityHandler());
     } else if (enableBearerAuth(authMethod)) {
       context.setSecurityHandler(createBearerSecurityHandler());
+    } else {
```

Comment:
I don't understand. What happens if you are using Basic Auth or Bearer Auth? The REJECT_OPTIONS_REQUEST config is ignored? Whoever uses this config is going to very security-aware, so they are probably going to be using authentication as well.

Reply:
Ow, you said you tested with Basic Auth and Bearer Auth. Sorry, couldn't find that piece of code. Let me look again, and please ignore. :)